### DISCUSSION TITLE
IDS App Store down?

### DISCUSSION THREAD

**Discussion by otmi100** [International-Data-Spaces-Association/DataspaceConnector#239] [Q&A] [answered]
The most recent [documentation](https://international-data-spaces-association.github.io/DataspaceConnector/CommunicationGuide/v6/IdsEcosystem/AppStore) says there should be a publicly available App Store:

> The IDS App Store is available at https://binac.fit.fraunhofer.de/. It expects IDS multipart messages at https://binac.fit.fraunhofer.de/api/ids/data. The GUI can be accessed at https://binac.fit.fraunhofer.de/appstore/. The access to all addresses is unrestricted.

Currently I'm getting a ERR_CONNECTION_CLOSED / PR_END_OF_FILE_ERROR / CONNECTION REFUSED error.

Is it just temporarily or permanently down?

**Comment by ghost** [comment]
Can't reach the FIT demo-AppStore either.
@ahemaid

**Reply by ahemaid** [reply]
@otmi100  and @tmberthold  
Yes, I am still working on the AppStore version which support DSC v.7. 
Therefore, from time to time, I am restarting it. 
The current version is supporting the self-description messages but still we are not able to download the meta-data of the apps on the new version of DSC. Maybe a meeting can help to finalize it.

**Comment by ahemaid** [comment, marked as answer]
@otmi100 @tmberthold 
Now the Appstore v.3  supports DSC v.7. 
It is working under Binac Appstore:
https://binac.fit.fraunhofer.de/api/ids/data.
